web_widget_url_advanced/templates/assets.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Copyright 2018 Simone Orsi - Camptocamp SA
+    License LGPLv3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html).
+-->
+<odoo>
```

Comment:
can you rebase on 13.0 and run pre-commit? There's a fix for bad xml formatting.